src/test/ui/issues/issue-59488.rs

```diff
@@ -8,19 +8,30 @@ fn bar(a: i64) -> i64 {
     43
 }
 
+enum Foo {
+    Bar(usize),
+}
+
 fn main() {
     foo > 12;
-    //~^ ERROR 12:9: 12:10: binary operation `>` cannot be applied to type `fn() -> i32 {foo}` [E0369]
-    //~| ERROR 12:11: 12:13: mismatched types [E0308]
+    //~^ ERROR 16:9: 16:10: binary operation `>` cannot be applied to type `fn() -> i32 {foo}` [E0369]
```

Comment:
You could have moved the definition of the enum to the bottom and therefore prevent changing all the linenumbers.

Reply:
I didn't know. Thank you!

Reply:
I don't know if that works what you did (I don't have much experience with the test framework). I meant, that instead of writing `enum foo { ... } fn main() { ... }` you could've done `fn main() { ... } enum foo { ... }`.

Reply:
Maybe it works. I read this commit.
https://github.com/rust-lang/rust/commit/8f77a035a43158d6d997d6c71e36e4a2c7eb7336#diff-64ce4eb189ca06032c10f8fbc3a5a62aL13

Reply:
Travis will tell us ;)

Reply:
🎉